about.html

```diff
@@ -0,0 +1,168 @@
+                <p class="logo-name">
+                    Matrix tech
+                </p>
+            </a>
+            <ul class="menu-items">
+                <li class="menu-item">
+                    <a href="about.html">About</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">Services</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">Clients</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">Contacts</a>
+                </li>
+                <li class="compaign">
+                    <button type="button" class="btn-compaign">
+                        Partcipate on projects
+                    </button>
+                </li>
+            </ul>
+        </div>
+        <div id="mobile-main-menu" class="overlay">
+            <div class="closebtn">&times;</div>
+            <div class="overlay-content">
+                <a href="about.html">About</a>
+                <a href="">Services</a>
+                <a href="#">Clients</a>
+                <a href="#">Contact</a>
+                <button type="button" class="btn-compaign">
+                    Partcipate on projects
+                </button>
+                <div class="bottom-right">
+                    <ul class="social-medias">
+                        <li class="social-media-item"><i class="fab fa-facebook fa-1x"></i></li>
+                        <li class="social-media-item"><i class="fab fa-google fa-1x"></i></li>
+                        <li class="social-media-item"><i class="fab fa-twitter fa-1x"></i></li>
+                        <li class="social-media-item"><i class="fab fa-youtube fa-1x"></i></li>
+                        <li class="social-media-item"><i class="fab fa-instagram fa-1x"></i></li>
+                        <li class="social-media-item">English</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <section id="about-main">
+        <h1 class="about-text-2 color-red cocogoose">
+            You share your idea.
+            We get it done
+        </h1>
+        <p class="about-text-3 my-20 font-light coocgoose">
+            We bring the technical expertise and develop the domain expertise to complement yours; we handle the
+            technical side of things while you focus on expanding your impact and realizing your mission.
+
+            As your technical partners, we demystify tech and neatly sidestep risk, ensuring that you achieve your goals
+            on time and on budget. We also help you navigate strategy, branding, and how to tell your story.
+        </p>
+        <p class="font-medium">
+            please contact us trough email for further questions and how to make your software applicable
+        </p>
+        <p class="font-bold font-medium">
+            metrixtech3@gmail.com
+        </p>
+    </section>
```

Comment:
- [ ] **Required:** Kindly fix the console errors on this page as well.

![image](https://user-images.githubusercontent.com/27709832/152436173-5fabd34b-4624-4743-8334-bb8356b9178c.png)

- [ ] **Required:** To have a similar design, consider the following:

1. The gap (padding/margin) given between the top of the page and the header is too big.
2. Use appropriate `font-family` and `font-weight`. Align the header to the centre on all screens of the `About` page.
3. The text isn't visible.
4. The mail is **_underlined_** and has a small arrowhead icon.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/152437445-242d9033-f712-4f52-b627-4a070ca5f872.png) | ![image](https://user-images.githubusercontent.com/27709832/152437525-aaa6f951-caf7-4586-b9b3-f6c28b597b5e.png)